logout.html

```diff
@@ -0,0 +1,103 @@
+!
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <!DOCTYPE html>
+  <html lang="en">
+
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>shoeKart</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Arimo&family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://aquaui.netlify.app/style.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+</head>
+
+
+<body class="page-container logout">
+  <div class="header">
+    <div class="header">
+      <div class="nav">
+        <div class="nav-left">
+          <a href="/index.html"> <img class="hero-logo" src="./images/running-shoes.png" alt=""></a>
+          <input class="search-bar" type="text" placeholder="Search..">
+          <a href="./login.html">
+            <div class="button button-icon"> <span> Login </span>
+            </div>
+          </a>
+        </div>
+
+        <div class="nav-right">
+          <figure class="icon icon-cart ">
+            <div> <i class="fa-solid fa-cart-shopping"></i></div>
```

Comment:
why have you put them in div?

Reply:
I had made this commit in dev, will rectify it